### DISCUSSION TITLE
ssr plus 的 v2ray 太大了，如何压缩？

### DISCUSSION THREAD

**Discussion by macaujack** [coolsnowwolf/lede#12156] [Q&A / 快速问答]
用的 Phicomm k2p，闪存大小只有 16 MB，求解如何压缩 ssr plus 的 v2ray？现在我 `make menuconfig` 里的 LuCI 的 Application 只选了 ssrplus 和 firewall 都无法生成 sysupgrade.bin。之前用过别人编译的固件，里面有支持所有节点类型的 ssr plus，不知道为什么自己编译就会太大。网上搜了一圈，好像可以用 upx 来压缩，但是具体不知道该如何操作。

**Comment by Cathgao** [comment]
您是哪怕50包邮的一个32m flash的路由器都买不起吗？16m 8m的设备别浪费大家时间了

**Reply by macaujack** [reply]
但是我真的好好奇为什么别人行，我不行…

**Comment by macaujack** [comment]
找到了恩山的[这个贴](https://www.right.com.cn/forum/thread-8030846-1-1.html)，2022 年的，说新版本 OpenWRT 的 WiFi 驱动变大了很多，不知道跟这个有无关系？若有，可否在 `make menuconfig` 中选择使用更小的 WiFi 驱动？

**Reply by Gokou-Ruri** [reply]
> 找到了恩山的[这个贴](https://www.right.com.cn/forum/thread-8030846-1-1.html)，2022 年的，说新版本 OpenWRT 的 WiFi 驱动变大了很多，不知道跟这个有无关系？若有，可否在 `make menuconfig` 中选择使用更小的 WiFi 驱动？

你把源码里的驱动给替换了不就行了吗？
